src/NUnitTestAdapter/AdapterSettings.cs

```diff
@@ -123,6 +125,18 @@ public void Load(string settingsXml)
 #endif
         }
 
+        public void SaveRandomSeed(string dirname)
+        {
+            using (var writer = new StreamWriter(Path.Combine(dirname, "$RANDOM_SEED$")))
```

Comment:
This and the read below should probably be wrapped in a try/catch. Too much can go wrong accessing the filesystem and we don't want to crash the adapter or fail the entire test run. Better to output a log message.